src/components/StudyCreate/StudyCreateMbtiSelect/MbtiBox/MbtiBox.tsx

```diff
@@ -0,0 +1,20 @@
+import { FilledMbtiBox, OutlinedMbtiBox } from './style';
+
+interface Props {
+  mbti: string;
+  filled?: boolean;
+  disabled?: boolean;
+}
+
+function MbtiBox({ mbti, filled, disabled }: Props) {
+  if (filled) return <FilledMbtiBox mbti={mbti}>{mbti}</FilledMbtiBox>;
+  else {
+    return (
+      <OutlinedMbtiBox mbti={mbti} disabled={disabled}>
+        {mbti}
+      </OutlinedMbtiBox>
+    );
+  }
```

Comment:
삼항 연산자로 처리하셔도 될 것 같습니다!
```suggestion
  return filled ? (
    <FilledMbtiBox mbti={mbti}>{mbti}</FilledMbtiBox>
  ) : (
    <OutlinedMbtiBox mbti={mbti} disabled={disabled}>
      {mbti}
    </OutlinedMbtiBox>
  );
```